config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml

```diff
@@ -0,0 +1,24 @@
+presubmits:
```

Comment:
what about the rest of the presubmits?

Reply:
do we want to for a N-2 release?

Reply:
fine by me, unless we plan to have support for 0.1 today ;P  jejeje 
to that point, do we state in the docs about support policy (N-1 stuff)

Reply:
We don't.
Unofficially, we just plan to support 0.3+, unless someone comes knocking the door for a fix in 0.2.
But 0.1 is old enough that I don't see it happening.